frameworks/Clojure/duct/src/hello/boundary/world_db.clj

```diff
@@ -0,0 +1,32 @@
+(ns hello.boundary.world-db
+  (:require [duct.database.sql]
+            [duct.database.mongodb.monger]
+            [clojure.java.jdbc :as jdbc]
+            [monger.collection :as mc]))
+
+(defn- query [db]
+  (first
+    (jdbc/query db ["select * from world where id = ?" (inc (rand-int 4))])))
+
+(defn- mongo-query [db]
+  (dissoc (mc/find-one-as-map db "world" {:id 5}) :_id))
```

Comment:
This looks like it's hardcoded to find the world with id=5.  It should be a randomly-generated id just like in the SQL version.